packages/components/src/molecules/SearchTop/SearchTopTerm.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { forwardRef } from 'react'
+import type { HTMLAttributes } from 'react'
+
+import { Badge, Link, LinkProps, LinkElementType } from '../../'
+
+export interface SearchTopTermProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+  /**
+   * Defines the text displayed in top term item.
+   */
```

Comment:
We can also make this clearer. I also think that "top term" here might be confusing. How about:

```suggestion
  /**
   * Search term to be shown.
   */
```